Cargo.toml

```diff
@@ -160,6 +160,7 @@ utilities = [ "snarkvm-utilities" ]
 wasm = [ "snarkvm-wasm" ]
 test_exports = [ "snarkvm-algorithms/test_exports" ]
 test_targets = [ "snarkvm-console/test_targets" ]
+test_consensus_versions = [ "snarkvm-console/test_consensus_versions" ]
```

Comment:
I think `test_consensus_heights` is a better name, because this feature changes the **height** at which a version is activated.